### ISSUE TITLE
Could not fetch package info

### ISSUE THREAD

**Issue by shuvashish76** [MuntashirAkon/AppManager#1121]
### Please check before submitting an issue

- [X] I know what my device, OS and App Manager versions are
- [X] I know how to take logs
- [X] I know how to reproduce the issue which may not be specific to my device

### Describe the bug

Can't update or even fresh install latest debug version.  `AM Debug v3.2.0-alpha01 Run#2505`
It shows this message "Could not fetch package info" when open with App Info or Install.
Explore option causes crashes.

### Logs

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{io.github.muntashirakon.AppManager.debug/io.github.muntashirakon.AppManager.fm.FmActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.net.Uri.getScheme()' on a null object reference
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3751)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3919)
    at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
    at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2311)
    at android.os.Handler.dispatchMessage(Handler.java:111)
    at android.os.Looper.loopOnce(Looper.java:238)
    at android.os.Looper.loop(Looper.java:357)
    at android.app.ActivityThread.main(ActivityThread.java:8090)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1026)
 Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.net.Uri.getScheme()' on a null object reference
   at io.github.muntashirakon.AppManager.fm.FmActivity.onAuthenticated(FmActivity.java:83)
   at io.github.muntashirakon.AppManager.BaseActivity.onCreate(BaseActivity.java:88)
   at android.app.Activity.performCreate(Activity.java:8232)
   at android.app.Activity.performCreate(Activity.java:8212)
   at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1331)
   at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3724)
   at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3919)
   at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
   at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
   at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
   at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2311)
   at android.os.Handler.dispatchMessage(Handler.java:111)
   at android.os.Looper.loopOnce(Looper.java:238)
   at android.os.Looper.loop(Looper.java:357)
   at android.app.ActivityThread.main(ActivityThread.java:8090)
   at java.lang.reflect.Method.invoke(Native Method)
   at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1026)
```

### Device info

App version: 3.2.0-alpha01-DEBUG
App version code: 433
Android build version: 8049b-fa5cf
Android release version: 12
Android SDK version: 31
Android build ID: S2RUBS32.51-15-9-7
Device brand: motorola
Device manufacturer: motorola
Device name: corfur
Device model: moto g71 5G
Device product name: corfur_g
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
System language: en-IN
In-App Language: auto
Mode: auto
Inferred Mode: No root

**Comment by MuntashirAkon** [comment]
Debug build 2504 had an issue with explorer which were fixed in the next build. If you can't install it via AM Debug even after clearing cache, you can try the native installer. Debug builds are super unstable and now and then will fail.

**Comment by shuvashish76** [comment]
Clearing cache of previous Debug build : Same issue
Latest Stable (3.1.1) version : Same but it doesn't crash with "Explore"
Native installer : _There was a problem while praising the package._

**Comment by MuntashirAkon** [comment]
Ah, you were running A12, I thought you were still testing on Android 7. Fixed in 70211eb85cba62b270fb45e5ae1652fc0a858325
